### ISSUE TITLE
400: Bad Request error

### ISSUE THREAD

**Issue by Portalboy** [kyleboe/lita-discord#1]
![badrequestlita](https://cloud.githubusercontent.com/assets/6773480/21819244/8358e134-d730-11e6-82c1-8663dc7c2bb1.png)
Not sure what I'm doing wrong, but I'm getting a 400: Bad Request when running Lita with the discord adapter.

I have the following config switches:
```  
  config.robot.adapter = :discord
  config.adapters.discord.email = "stuff@things.com"
  config.adapters.discord.password = "secret"
```

**Comment by kyleboe** [comment]
Yeah this is very WIP, I have a little bit of extra time on the weekends so I will start working on it again. I'll be sure to let you know when it is remotely functional 😅
